app/components/inplace-select.js

```diff
@@ -1,6 +1,8 @@
 import Ember from 'ember';
 import InPlace from 'ilios/mixins/inplace';
 
+const { observer } = Ember;
+
 export default Ember.Component.extend(InPlace, {
```

Comment:
I don't think inPlace select is the place for this.  InPlaceSelect is used when we want to put a checkbox next to an item and save it immediately.  These should just be regular select lists because the save happens when the continue button is clicked.